crates/valence_inventory/src/player_inventory.rs

```diff
@@ -16,6 +12,7 @@ impl PlayerInventory {
     pub const SLOT_CRAFT_RESULT: u16 = 0;
     pub const SLOTS_HOTBAR: RangeInclusive<u16> = 36..=44;
     pub const SLOTS_MAIN: RangeInclusive<u16> = 9..=44;
+    pub const MAIN_SIZE: u16 = 36;
```

Comment:
Can you have `MAIN_SIZE` be derived from `SLOTS_MAIN`?

Reply:
I'm not sure how. It needs to be in a const context

Reply:
That's perfect. I'm kinda surprised there's not a `len()` on `RangeInclusive` though